mtbf_driver/tests/mtbf/browser/test_mtbf_browser_clear_browsing_history.py

```diff
@@ -48,5 +48,6 @@ def test_browser_clear_history(self):
         self.assertEqual(0, search.history_items_count)
 
     def tearDown(self):
+        self.apps.kill(self.settings.app)
```

Comment:
in MTBF, the apps shouldn't be killed too often. How about try to recover it back to the initial screen of settings app?
https://github.com/Mozilla-TWQA/MTBF-Driver/blob/master/mtbf_driver/mtbf_apps/settings/app.py#l31